src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -739,6 +735,7 @@ def __write_status_file(self):
         self.env_layer.file_system.write_with_retry_using_temp_file(self.complete_status_file_path, '[{0}]'.format(status_file_payload_json_dumps), mode='w+')
 
         if Constants.StatusTruncationConfig.TURN_ON_TRUNCATION:
+            self.composite_logger.log_debug("Check status file to perform truncation")
```

Comment:
This log again suggests that we are only checking the status file for truncation.

change to: self.composite_logger.log_debug("Perform truncation on status file if applicable")

Reply:
addressed